odkx-src/notify-using.rst

```diff
@@ -0,0 +1,147 @@
+Using ODK-X Notify
+=======================
+
+Skunkworks-Parrot (Desktop App):
+---------------------------------
+
+Creating Notification:
+"""""""""""""""""""""""
+
+1. Launch the Desktop application(Skunkworks-Parrot)
+2. Enter the Username and Password in the provided field and click :menuselection:`Login`.
+  
+  .. image:: /img/notify-using/notify-login.*
+
+.. note::
+
+   Only users with the default group “ROLE_SITE_ACCESS_ADMIN” are allowed to log in to the Desktop application.
```

Comment:
I think this needs a little more explanation. Does this refer to the standard ODK-X permission filters? If so, those should be cross-referenced. If not, please explain.

Reply:
Yes. This is referring to [ODK-X permission filters](https://docs.odk-x.org/data-permission-filters/#obtaining-a-user-s-default-group). But in the permission filter page, I didn't find anything related to ROLE_SITE_ACCESS_ADMIN. So not sure where to cross-reference it.

Reply:
Is this something that then needs to be documented in the permission filters? @jbeorse or others do you know?

Reply:
To be honest I am not totally sure about that role. I think the reason this isn't included there is that this role isn't to do with filtering date so much as it is access to different tools. We don't have this documented well in the docs right now. In my opinion we should call this out clearly hear and then separately think about how we can update the docs to more clearly describe all possible roles and how they work (along with how creating your own roles works). I'd want to loop @linl33 into that conversation. 

Personally I would not block this PR on this issue but I defer to @elmps2018 if she disagrees